examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,76 @@
+"""
+Multi-subject connectome for ADHD
+=================================
+
+This example estimates a connectome from spheres in the brain. The coordinates
+of the spheres have been taken from a meta-analysis on neurosynth website.
+
+"""
+import matplotlib.pyplot as plt
+
+
+# Fetching datasets ###########################################################
+print("-- Fetching datasets ...")
+from nilearn import datasets
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+
+# print basic information on the dataset
+print('First subject functional nifti image (4D) is at: %s' %
+      adhd_dataset.func[0])  # 4D data
+
+
+# Extracting region signals ###################################################
+from nilearn import image
+from nilearn import input_data
+
+from sklearn.externals.joblib import Memory
+mem = Memory('nilearn_cache')
+
+print("... Extracting time series ...")
+
+# Coordinates of Default Mode Network
+dmn_coords = [(0, -52, 18), (-46, -68, 32), (46, -68, 32), (0, 50, -5)]
+labels = [
+    'Posterior Cingulate Cortex',
+    'Left Temporoparietal junction',
+    'Right Temporoparietal junction',
+    'Medial prefrontal cortex'
+]
+# colors = plt.cm.gist_rainbow(np.linspace(0, 1, len(dmn_coords)))
+colors = ['b', 'g', 'r', 'm']
```

Comment:
Can't you make the example simpler by not specifying colors and letting matplotlib choose some?